cranelift/filetests/filetests/isa/aarch64/arithmetic.clif

```diff
@@ -342,11 +342,17 @@ block0(v0: i8x16):
   return v2
 }
 
+;   stp fp, lr, [sp, #-16]!
```

Comment:
PR #4521 should remove these unnecessary instructions.

Reply:
👍 Will rebase once that lands